src/bucket/FutureBucket.h

```diff
@@ -59,7 +59,8 @@ class FutureBucket
     std::shared_ptr<Bucket> mInputCurrBucket;
     std::shared_ptr<Bucket> mInputSnapBucket;
     std::vector<std::shared_ptr<Bucket>> mInputShadowBuckets;
-    std::shared_future<std::shared_ptr<Bucket>> mOutputBucket;
+    std::shared_ptr<Bucket> mOutputBucket;
+    std::shared_future<std::shared_ptr<Bucket>> mOutputBucketFuture;
```

Comment:
no test update?

Reply:
We do not test the internals of FutureBucket, only its public interface, which functionally did not change.